viewmaster/viewmaster.py

```
@@ -49,33 +70,58 @@
+    Sets up a PdsLogger instance with INFO and DEBUG level logging to files
+    with midnight rotation. If running in an interactive terminal, also adds
+    stdout handler. Configures Pds3File to use this logger.
+
+    Returns:
+        pdslogger.PdsLogger: Configured logger instance.
+    """
+
+    global LOGGER
+
+    try:
+        logger = pdslogger.PdsLogger.get_logger(LOGNAME)
```

Ruff (0.14.10)

`LOGNAME` may be undefined, or defined from star imports

(F405)